pyproject.toml

```
@@ -1,3 +1,44 @@
+[build-system]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
```

I'd say we need 
```suggestion
requires = ["setuptools>=70.1", "setuptools_scm[toml]>=6.2"]
```

To make sure `wheel` is around. xref https://github.com/conda-forge/pip-feedstock/issues/124#issuecomment-2261091498